sqlitecluster/SQLiteNode.cpp

```diff
@@ -67,7 +67,7 @@ const string SQLiteNode::consistencyLevelNames[] = {"ASYNC",
 
```

Comment:
NAB the comment at the top of this file is slightly out of date now since it talks about `STCPNode` https://github.com/Expensify/Bedrock/blob/693f220a5fd43f31713ead7aa101b1e4e70d1398/sqlitecluster/SQLiteNode.cpp#L13-L15